source/agg-pipeline-builder/view-pipeline-explain-plan.txt

```diff
@@ -0,0 +1,74 @@
+.. _view-pipeline-explain-plans:
+
+=================================
+View Explain Plans for a Pipeline
+=================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To help you better understand the performance of your pipeline, you can view 
+your pipeline's explain plan. You can view the explain plan at any point while 
+creating or editing your pipeline.
+
+About this Task
+---------------
+
+On the :guilabel:`Explain` modal, you can view the explain stages as a 
+:guilabel:`Visual Tree`, where each stage of the pipeline appears as a node on 
+the tree. Alternatively, you can view the explain details in raw JSON format. 
+
+The explain plan includes a :guilabel:`Query Performance Summary` with 
+information on the execution of your aggregation pipeline such as: 
+
+- Execution time. 
+
+- The number of returned documents.
+
+- The number of examined documents.
+
+- The number of examined index keys.
+
+Procedure
+---------
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: Click the :guilabel:`Explain` button.
```

Comment:
([titles](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#style-and-structure))
You can omit font treatment from headings and titles.